### PR TITLE
Revert "added --user to installs"

### PR DESCRIPTION
**Pull request by PiotrJander**
This reverts commit 14967f60f8966fab3f4ca268d6e21d26af61ca6c.

### CODE REVIEW

**Comment by jakubczakon** [comment]
Mhm, now it only faisl on windows due to failed (misplaced) tensorflow install. 

```
ERROR: Could not install packages due to an EnvironmentError: [WinError 5] Access is denied: 'c:\\hostedtoolcache\\windows\\python\\3.6.8\\x64\\lib\\site-packages\\~umpy\\.libs\\libopenblas.NOIJJG62EMASZI6NYURL6JBKM4EVBGM7.gfortran-win_amd64.dll'
2020-10-26T12:11:35.4549200Z 
2020-10-26T12:11:35.4549709Z Consider using the `--user` option or check the permissions.
```

**Comment by PiotrJander** [comment]
Yeah, I'm not user `pip install --user` is a perfect idea, especially since it doesn't work in virtual environments.

So it is left to debug why tensorflow fails - or simply Tensorflow on Windows, where this happens, until we know what causes Tensorflow to break on Windows.

File: ci/test_examples.py [hunk 1]
```diff
@@ -6,6 +8,24 @@
 test_files = glob('**/tests/*.py', recursive=True)
 
 excluded_files = []
+if os.name == 'nt': # if OS is Windows
+    # Excluding because unable to install Tensorflow on a Windows CI server with
+    #
+    # ERROR: Could not install packages due to an EnvironmentError: [WinError 5] Access is denied: 'c:\\hostedtoolcache\\windows\\python\\3.6.8\\x64\\lib\\site-packages\\~umpy\\.libs\\libopenblas.NOIJJG62EMASZI6NYURL6JBKM4EVBGM7.gfortran-win_amd64.dll'
+    # Consider using the `--user` option or check the permissions.
+    excluded_files.extend(glob('product-tours\\how-it-works\\tests\\*.py', recursive=True))
```
Comment (jakubczakon) [inline ci/test_examples.py:16]: So we are not testing this example at all on windows -> we should add this info to the readme @kamil-kaczmarek @PiotrJander

File: ci/test_examples.py [hunk 2]
```diff
@@ -6,6 +8,24 @@
...
+    # Excluding because on a Windows CI server with Python 3.8, tkinter error occurs.
+    #
+    # Traceback (most recent call last):
+    #
+    #   File "c:\hostedtoolcache\windows\python\3.8.6\x64\lib\tkinter\__init__.py", line 4014, in __del__
+    #
+    #     self.tk.call('image', 'delete', self.name)
+    #
+    # RuntimeError: main thread is not in main loop
+    excluded_files.append('integrations\\xgboost\\tests\\Neptune-XGBoost_upgraded_libs.py')
```
Comment (jakubczakon) [inline ci/test_examples.py:27]: This also should be added to the readme and the docs for XGBoost integration explicitly @kamil-kaczmarek .

Comment (kamil-kaczmarek) [reply ci/test_examples.py:30]: It's already added to the docs. Waiting for your review :) Take a look here: https://github.com/neptune-ai/docs/pull/123

Comment (kamil-kaczmarek) [reply ci/test_examples.py:30]: Which readme you have in mind?

Comment (jakubczakon) [reply ci/test_examples.py:30]: Readme of this repo (neptune-examples), in case we were updating things later.

File: ci/test_examples.py (hunk 1 above)
Comment (kamil-kaczmarek) [reply ci/test_examples.py:17]: @PiotrJander already changed tihs to test on py 3.8

File: ci/test_examples.py (hunk 2 above)
Comment (kamil-kaczmarek) [reply ci/test_examples.py:30]: added.

File: .github/workflows/neptune-examples.yml [hunk 3]
```diff
@@ -1,6 +1,8 @@
 name: neptune-examples
 on:
   push:
+    paths-ignore:
+    - 'README.md'
```
Comment (jakubczakon) [inline .github/workflows/neptune-examples.yml:5]: Awesome thought, was going to suggest this!

File: README.md [hunk 4]
```diff
@@ -127,14 +127,15 @@ For example:
...
+    - Neptune-API-Tour.ipynb  
+    - Organize-ML-experiments.ipynb        
+    - r-integration.ipynb  
+    - Use-Neptune-API-to-log-your-first-experiment.ipynb
+    - Monitor-ML-runs-live.ipynb  
+    - neptune_test_run.ipynb  
+    - pytorch_lightning-integration.ipynb  
+    - Template.ipynb       
+    - xgboost-integration.ipynb
+- XGBoost integration (upgraded libs case) is not tested on Win with Python 3.8, as it causes [tkinter error](https://github.com/neptune-ai/neptune-examples/runs/1309037471?check_suite_focus=true).
```
Comment (jakubczakon) [inline README.md:141]: @kamil-kaczmarek we should mention why the "how-it-works product tour" was excluded in a similar fashion.

Comment (kamil-kaczmarek) [reply README.md:141]: done.

**Pull request merged**